packages/build/src/publish-mongosh.ts

```diff
@@ -22,6 +23,7 @@ export async function publishMongosh(
   barque: Barque,
   createAndPublishDownloadCenterConfig: typeof createAndPublishDownloadCenterConfigFn,
   publishToNpm: typeof publishToNpmType,
+  pushTags: typeof pushTagsType,
```

Comment:
these helper functions are getting out of hand and only seem to exist for the purpose of spying/stubbing with sinon. I am keeping same structure here but think it'd be good to wrap them in a namespace or something in the future.